marklogic-data-hub/src/main/resources/ml-modules/root/data-hub/5/impl/hub-utils.sjs

```diff
@@ -113,19 +113,26 @@ class HubUtils {
   }
 
   invoke(moduleUri, parameters, user = null, database) {
+    let options = buildInvokeOptions(user, database);
+    xdmp.invoke(moduleUri, parameters, options)
+  }
+
+  buildInvokeOptions(user = null, database) {
     let options = {
       ignoreAmps: true
     };
 
-    if(user && user !== xdmp.getCurrentUser()) {
+    if (user && user !== xdmp.getCurrentUser()) {
       options.userId = xdmp.user(user);
     }
-    
-    if(database) {
-      options.database = database ? xdmp.database(database): xdmp.database();
+
+    if (database) {
```

Comment:
I removed the "database ?" check here too.

Reply:
guess if no database is specified, the default is current database without requiring to set options.database ?

Reply:
correct, its assumed if nothing is provided, its the current db